pom.xml~

```diff
@@ -0,0 +1,121 @@
+ <project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Please remove all backup files (end in ~)

Reply:
Removed file in 3e571b0